docs/tutorials/cloud_fit_deploy/cloud-aws-sagemaker-deployment.md

```diff
@@ -0,0 +1,216 @@
+# Deploying AutoGluon Models with AWS SageMaker
+:label:`cloud_aws_sagemaker_deploy`
+
+After learning how to train a model using AWS SageMaker :ref:`cloud_aws_sagemaker_fit`, in this section we will learn how to deploy 
+trained models using AWS SageMaker and Deep Learning Containers. 
+
+The full end-to-end example is available in [amazon-sagemaker-examples](https://github.com/aws/amazon-sagemaker-examples/tree/master/advanced_functionality/autogluon-tabular-containers) repository.
+
+## Pre-requisites
+Before starting ensure that the latest version of sagemaker python API is installed via (`pip install --upgrade sagemaker`). 
+This is required to ensure the information about newly released containers is available.
+
+## Endpoint Deployment - Inference Script
+
+To start using the containers, an inference script and the [wrapper classes](https://github.com/aws/amazon-sagemaker-examples/blob/master/advanced_functionality/autogluon-tabular-containers/ag_model.py) are required.
+When authoring an inference [scripts](https://github.com/aws/amazon-sagemaker-examples/blob/master/advanced_functionality/autogluon-tabular-containers/scripts/), 
+please refer to SageMaker [documentation](https://docs.aws.amazon.com/sagemaker/latest/dg/adapt-inference-container.html).
+
+Here is one of the possible inference scripts. 
+
+- the `model_fn` function is responsible for loading your model. It takes a `model_dir` argument that specifies where the model is stored. 
+
+- the `input_fn` function is responsible for deserializing your input data so that it can be passed to your model. It takes input data and 
```

Comment:
I feel we should describe either transform_fn or (input_fn, predict_fn and output_fn). A link to the documentation might be helpful: https://docs.aws.amazon.com/sagemaker/latest/dg/neo-deployment-hosting-services-prerequisites.html

Reply:
Good catch. It supposed to be `transform_fn` - fixed.